### DISCUSSION TITLE
Pause Nova Resource Polling

### DISCUSSION THREAD

**Discussion by lonnylot** [laravel/nova-issues#5892] [Ideas & Feature Requests] [answered]
It's impossible to click multiple checkboxes and scroll up to the 'action' dropdown when resource polling is set because the entire screen refreshes in the middle of your work and clears all the checkboxes.

Is it possible to pause polling automatically when a resource is checked on the index page?

**Comment by milewski** [comment, marked as answer]
Supported on the latest Nova version :D

https://nova.laravel.com/releases/4.27.13

**Reply by lonnylot** [reply]
Been awhile! Thanks for monitoring and letting me know 🙏
